game/static/game/css/coming_soon.css

```
@@ -0,0 +1,147 @@
+}
+
+.crest {
+    width: 70px;
+    height: auto;
+    margin-bottom: 0.85rem;
+    filter: brightness(1.5) contrast(1.2) drop-shadow(0 0 8px rgba(240, 192, 64, 0.35));
+    animation: float 4s ease-in-out infinite;
+}
+
+.feature-label {
+    font-family: 'Cinzel', serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)